src/lib/hooks/useEip1193Provider.ts

```diff
@@ -17,32 +16,42 @@ class Eip1193Bridge extends ExperimentalEip1193Bridge {
         const result = await this.provider.getNetwork()
         return '0x' + result.chainId.toString(16)
       }
+      case 'eth_sendTransaction': {
+        if (!this.signer) break
+
+        // TODO(zzmp): JsonRpcProvider filters from/gas fields from the params.
```

Comment:
This should be opened as an issue on github.com/ethers-io/ethers.js, and someone else should verify this as a bug.